server/routes/trial-management/add-panel-members/add-panel-member.controller.js

```diff
@@ -0,0 +1,199 @@
+(function() {
+  'use strict';
+
+  const  _ = require('lodash')
+    , addPanelMembersValidator = require('../../../config/validation/add-panel-members')
+    , { addPanelMembersDAO, availableJurorsDAO } = require('../../../objects/panel')
+    , poolsValidator = require('../../../config/validation/generate-panel-pools')
+    , validate = require('validate.js');
+
+  const countErrors = (tmpErrors) => typeof tmpErrors !== 'undefined' ? Object.keys(tmpErrors).length : 0;
+
+
+  module.exports.getAddPanelMember = function(app) {
+    return function(req, res) {
+      let tmpErrors
+        , tmpFields;
+
+      tmpErrors = _.clone(req.session.errors);
+      tmpFields = _.clone(req.session.formFields);
+      delete req.session.errors;
+      delete req.session.formFields;
+
+      return res.render('trial-management/add-panel-members/index.njk', {
+        processUrl: app.namedRoutes.build('trial-management.add-panel-members.post', {
+          trialNumber: req.params.trialNumber,
+          locationCode: req.params.locationCode,
+        }),
+        cancelUrl: app.namedRoutes.build('trial-management.trials.detail.get', {
+          trialNumber: req.params.trialNumber,
+          locationCode: req.params.locationCode,
+        }),
+        panelDetails: tmpFields,
+        errors: {
+          count: countErrors(tmpErrors),
+          items: tmpErrors,
+        },
+      });
+    };
+  };
+
+  module.exports.postAddPanelMember = function(app) {
+    return function(req, res) {
+      let validatorResult;
+
+      validatorResult = validate(req.body, addPanelMembersValidator());
```

Comment:
this could be `const validatorResult = ....`

Reply:
Changed